.dockstore.yml

```diff
@@ -111,3 +111,6 @@ workflows:
   - name: DetectPCANovelties
     subclass: WDL
     primaryDescriptorPath: /Utilities/WDLs/DetectPCANovelties.wdl
+  - name: FindSamplesAndSimpleBenchmark
```

Comment:
Should we remove `FindSamplesAndBenchmark` from dockstore at this point? Or are there still features one might choose to use the old version for?

Reply:
The plan is to eventually replace it completely with this WDL. It would have to be done at the same time as replacing BenchmarkVCFs => SimpleBenchmark code, so this is more a "staging" PR and that merge will happen after checking with enough people that breaking main won't ruin all their dockstore imports (they should pin the version).